inception-project/inception
# #2501 - Include layer and value names in XMI export

**What's in the PR**
- Reorganizing export code into the export module
- Starting towards including the layer/feature info into the CAS as part of the export

**How to test manually**
* Export an annotated file as XMI and check the XMI file for the layer definitions.

**Automatic testing**

**Documentation**
